bigchaindb_driver/driver.py

```diff
@@ -27,7 +27,8 @@ def __init__(self, *nodes, transport_class=Transport, headers=None):
             headers (dict): Optional headers that will be passed with
                 each request. To pass headers only on a per-request
                 basis, you can pass the headers to the method of choice
-                (e.g. :meth:`~.TransactionsEndpoint.send`).
+                (e.g. :meth:`BigchainDB().transactions.send()
```

Comment:
I find the `BigchainDB()` a little bit confusing but having just `.transactions.send()` is only marginally better.